pyang/plugins/tree.py

```diff
@@ -144,14 +144,12 @@ def print_header(module):
             if b is not None:
                 bstr = " (belongs-to %s)" % b.arg
             fd.write("%s: %s%s\n" % (module.keyword, module.arg, bstr))
-            printed_header.append(None)
```

Comment:
shouldn't this be `printed_header = True` ?

Reply:
Yep， you are right. Now the Github Action passed.

> shouldn't this be `printed_header = True` ?